### ISSUE TITLE
advice on multi-dataset training

### ISSUE THREAD

**Issue by lunixbochs** [facebookresearch/wav2letter#283]
I've managed to train a hybrid model on librispeech, Common Voice, and some other data successfully. This using the high dropout ASG conv_glu librispeech recipe. Adding in the Tedlium dataset seems to just cause problems and I'm really not sure why or how to debug it.

When I mix in the Tedlium dataset, the model immediately does worse at everything, including the Tedlium test data. The other tests only fluctuate slightly, like librispeech goes from ~TER 2.7 to 2.8, but removing Tedlium from the training data brought the Tedlium test TER from 90 down to 60 very quickly. I also noticed that the Tedlium TER fluctuates wildly when Tedlium is in the training set, but is very stable when it's only in the test set.

This is with a fairly low learning rate as well, I'm ~80 epochs in and my lr is 0.05 and lrcriterion is 0.001 right now.

What kind of parameters can I tweak? Is there anything I can check about the input data? Tedlium is stored as sph files while I've converted the rest to flac or wav, but I assume if there were problems reading the files it wouldn't be able to get a TER of 60 in test.

This is how I'm doing the data prep for Tedlium: https://github.com/talonvoice/speech/blob/master/projects/05-tedlium/1-data_prep.sh

This is how one of the lst files looks:
```
KakenyaNtaiya_2012X-1 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-1.sph 4120.00 in kenya people cross oceans
KakenyaNtaiya_2012X-2 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-2.sph 9980.00 to go see them these people are tall they jump high they wear red and they kill lions
KakenyaNtaiya_2012X-3 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-3.sph 3770.00 might be wondering who are these people these are the
KakenyaNtaiya_2012X-4 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-4.sph 5150.00 maasais and you know what's cool i'm actually one of them
KakenyaNtaiya_2012X-5 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-5.sph 8660.00 the maasais the boys are brought up to be warriors the girls are brought up to be mothers when i was
KakenyaNtaiya_2012X-6 /data/processed/tedlium/clips/KakenyaNtaiya_2012X-6.sph 4050.00 five years old i found out that i was engaged
```
